### DISCUSSION TITLE
stopwatch

### DISCUSSION THREAD

**Discussion by chipset2011** [zepp-health/zeppos-forum#345] [User Feedback]
Dear developers, I wrote about the stopwatch a long time ago, I kindly ask you to add start and stop controls to the top button, or at least to the bottom button, or better yet, add a new lap countdown to the bottom button. For everything to be like in current stopwatches, it is very inconvenient to poke on the touch screen.

**Comment by MyoungXUE** [comment]
Hello, What is your device?

**Reply by chipset2011** [reply]
> Hello, What is your device?

GTR4

**Reply by chipset2011** [reply]
> Hello, What is your device?

info?

**Reply by MyoungXUE** [reply]
Settings -> Preferences -> Button Mode

**Reply by MyoungXUE** [reply]
Sorry I just noticed this problem and kept you waiting.

**Reply by chipset2011** [reply]
thanks!

**Reply by chipset2011** [reply]
Settings -> Preferences -> Button Mode

and then can I add to the watch the ability to make the return to the home screen function on the button? For example, for a clamped button. I didn't see such a feature in the button settings.

**Reply by MyoungXUE** [reply]
Settings -> Preferences -> Long Press Upper Button
Settings -> Preferences -> Press Lower Button

This feature provides some shortcuts, you can have a try.
